src/components/common/ForeignJurisdiction.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-card flat class="pt-6 px-4">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3">
+        <label class="title-label">Jurisdiction</label>
+      </v-col>
+      <v-col cols="12" sm="9">
+        <v-select
+          id="country-selector"
+          :items="countryNames"
+          v-model="selectedCountry"
+          filled
+          placeholder="Jurisdiction Country"
+          :rules="countryRules"
+          @change="emitChangedCountry($event)"
+        />
```

Comment:
You'll have to tweak a style so the "g" shows correctly in this drop-down menu. (Probably ditto for jurisdiction province.)

I did this somewhere, some time. I can't remember. Just look at the various elements that Vuetify renders for this component and you'll find the one that is too short or whatever.

Reply:
I think same is id for "y". Fix for this might fix both

Reply:
Fix looks good:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/357a5e6d-dd00-4c09-9f0f-d28f756640bb)

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/2772b9af-e6cc-4eb0-bd96-4ca1d1eb4f7d)